src/common/snippets/include/snippets/op/subgraph.hpp

```diff
@@ -101,19 +101,11 @@ class Subgraph : public ov::op::util::SubGraphOp {
     bool is_quantized() const { return config.m_is_quantized; }
     bool has_type_relaxed_ops() const { return config.m_has_type_relaxed_ops; }
     bool has_domain_sensitive_ops() const { return config.m_has_domain_sensitive_ops; }
-    snippets::Schedule generate(const BlockedShapeVector& output_shapes,
-                                const BlockedShapeVector& input_shapes,
-                                ngraph::pass::Manager& pre_dialect,
-                                ngraph::pass::Manager& post_dialect,
-                                ngraph::pass::Manager& post_precision,
-                                const void* compile_params = nullptr);
-    snippets::Schedule generate(const BlockedShapeVector& output_shapes, const BlockedShapeVector& input_shapes, const void* compile_params = nullptr);
```

Comment:
Why did you remove common interface? Could you please elobarate?

Reply:
discussed, returned